data.sql

```diff
@@ -1,8 +1,18 @@
 /* Populate database with sample data. */
 
-INSERT INTO animals  (id, name, date_of_birth, escape_attempts, neutered, weight_kg) 
-VALUES (1, 'Agumon', '2020-2-3', 0, true, 10.23),
-  (2, 'Gabumon', '2018-11-15', 2, true, 8),
-  (3, 'Pikachu', '2021-1-7', 1, false, 15.04),
-  (4, 'Devimon', '2017-5-12', 5, true, 11);
+INSERT INTO animals  (name, date_of_birth, escape_attempts, neutered, weight_kg) 
+VALUES ('Agumon', '2020-2-3', 0, true, 10.23),
+  ('Gabumon', '2018-11-15', 2, true, 8.0),
+  ('Pikachu', '2021-1-7', 1, false, 15.04),
+  ('Devimon', '2017-5-12', 5, true, 11.0);
 
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) 
+VALUES ('Charmander', '02-08-2020', 0, false, -11.0),
+('Plantmon', '11-15-2021', 2, true, -5.7),
+('Squirtle', '04-02-1993', 3, false, -12.13),
```

Comment:
- [optional ] Great job making the changes.. i can see you are mixing up the date formats. kindly stick to the yy-mm-dd date format through out your project.